analysis/wwz/wwz4l.py

```diff
@@ -1045,9 +1076,28 @@ def process(self, events):
                             "category"      : sr_cat,
                             "systematic"    : wgt_fluct,
                         }
-                        hout[dense_axis_name].fill(**axes_fill_info_dict)
-
-        return hout
+                        self.accumulator[dense_axis_name].fill(**axes_fill_info_dict)
+
+            if self._siphon_bdt_data:
+                # Fill the list accumulator
+                self.accumulator["bdt_of_wwz_list"] += bdt_of_wwz[sr_4l_bdt_of_trn].to_list()
```

Comment:
Would it make sense to set this part up as a for loop so we don't have to do all of these lines of `+=`?

Reply:
how do you do that when the behaviors are different? if you do a for loop it's gonna have ugly if statements

Reply:
Hmm, it looks like most of the lines are fairly similar. OF and SF seem to do the same thing. I will try to think about it a bit more.